support-frontend/assets/components/sepaForm/SepaForm.tsx

```diff
@@ -27,11 +26,11 @@ export type SepaFormProps = {
 	iban?: string;
 	accountHolderName?: string;
 	addressStreetName?: string;
-	addressCountry?: Country;
+	addressCountry?: string;
```

Comment:
This is now a string since I deleted countries.ts - This definition of Country was only 
```typescript
export type Country = string;
```
anyway so no type safety is lost